db.go

```diff
@@ -80,7 +80,7 @@ func NewDB(dsn string) (*DB, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Could not create tables: %s", err)
 	}
-	reputationSelectStmt, err := db.Prepare("SELECT ip, reputation FROM reputation WHERE ip >>= $1 ORDER BY @ ip LIMIT 1;")
+	reputationSelectStmt, err := db.Prepare("SELECT ip, reputation FROM reputation WHERE ip >>= $1 LIMIT 1;")
```

Comment:
`ORDER BY @ ip` selects the smallest matching subnet. I think we may want it to stay that way?